rustls/src/webpki/verify.rs

```diff
@@ -47,14 +29,15 @@ pub fn verify_server_cert_signed_by_trust_anchor(
     roots: &RootCertStore,
     intermediates: &[Certificate],
     now: SystemTime,
+    supported_algs: &[&dyn webpki::SignatureVerificationAlgorithm],
```

Comment:
I think we wanted to avoid having items from the webpki crate in the rustls public API?

Reply:
Urgh, gross. I don't see a way of doing that without wrapping and reexporting every single `webpki::SignatureVerificationAlgorithm` value.

Reply:
Actually, such a wrapper would need to be constructable outside the crate (to address the webpki-without-ring use case) with a value that is `dyn webpki::SignatureVerificationAlgorithm` which would also count as a webpki type in our public API. I'm not sure this is actually possible, given the goal is to pass through an impl of a trait exported by the webpki crate?